persist/src/main/resources/db/migration/V11__UpdateContact.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE contacts ADD COLUMN referrer TEXT DEFAULT null;
```

Comment:
Is `DEFAULT null` required? Isn't null always the default?

Reply:
```suggestion
ALTER TABLE contacts 
ADD COLUMN referrer TEXT;
```
⛏️ (this means nitpick, not required but nice to have)

Reply:
⛏️ referral_source might be more clear than referrer